### ISSUE TITLE
As a publisher I want to be able to mark a Revision as "significant"

### ISSUE THREAD

**Issue by aschweigert** [Automattic/newspack-plugin#1732]
...and have this revision not automatically purged if we limit the number of revisions for performance reasons (see #1731)
...additionally the date/time for the last significant revision would be used as the "last updated date" on the post.

Discussion and further context: pamTN9-4Fd-p2#comment-6097

**Comment by gamebits** [comment]
@aschweigert Would you consider this feature request a blocker for https://github.com/Automattic/newspack-plugin/issues/1731?

**Comment by aschweigert** [comment]
Not necessarily, I think if we set the limit to something reasonably high and made sure the publisher knows the limit exists we'd be fine in almost all scenarios. This would just be a nice enhancement to have.
